README.md

```diff
@@ -34,6 +44,20 @@ cd ao
 pip install -e .
 ```
 
+## Our Goals
+torchao embodies PyTorch’s design philosophy [details](https://pytorch.org/docs/stable/community/design.html), especially "usability over everything else". Our vision for this repository is the following:
+
+* PyTorch Native: Provide PyTorch native solutions for optimization techniques for transformer models, offering composability with torch.compile and distributed 
+    * For example, for QLoRA for new dtypes support
+* Interoperability: Integration with other major transformer accel libraries in PyTorch enabling smooth user journey and accelerating visibility and adoption
+    * Examples - TorchTune, gpt-fast, ExecuTorch.
+* Transparent Benchmarks: Regularly run performance benchmarking of our APIs across a suite of Torchbench models and across hardware backends
+* Heterogeneous Hardware: Efficient kernels that can run on CPU/GPU based server (w/ torch.compile) and mobile backends (w/ ExecuTorch).
+* Autotuner: Invest in building a kernel auto tuner targeting SOTA performance across varying model shapes. 
```

Comment:
i'd move the autotuner to the features section as in it helps varying shapes and also both consumer and enterprise gpus